package.json

```diff
@@ -16,12 +16,25 @@
     "test-coveralls": "istanbul cover _mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | coveralls && rm -rf ./coverage",
     "upload-to-coveralls": "cat ./coverage/lcov.info | coveralls"
   },
+  "contributors": [
+    {
+      "name": "Andrew D.Laptev",
+      "email": "a.d.laptev@gmail.com",
+      "url": "https://github.com/agsh/onvif"
+    },
+    {
+      "name": "Jeff Galbraith",
+      "email": "jgalbraith@intelliviewtech.com",
+      "url": "http://intelliviewtech.com"
+    }
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/agsh/onvif.git"
   },
   "dependencies": {
-    "xml2js": "^0.4"
+    "request": "^2.83.0",
```

Comment:
See above about making request a dep.

Reply:
I like this approach a lot. It makes sense:

```
const onvif = require('onvif');
require('onvif-snapshot');
```

Requiring onvif-snapshot should automatically add a method to the `Cam` prototype just as if it was actually included in the library itself.

Reply:
I'm not exactly sure how to proceed to do this.

Reply:
I don't the ability to reply to your final question, so putting it in here:
1) Sure, close the PR.
2) I don't know how to cherry pick using Github. I'll see if I can figure it out.
3) Yes, create the repo please. If yu can add the initial code that binds it to onvif, that would be great and I'll fill in the body of code. I am not exactly sure how to do that part.
4) At some point in time, I'd like to rewrite the tests to use Jest, Chai and Nock (mocking server) if that's all right with you.